yarn-project/noir-compiler/package.json

```diff
@@ -40,7 +40,11 @@
     "@noir-lang/noir_wasm": "0.5.1-9740f54",
     "commander": "^9.0.0",
     "fs-extra": "^11.1.1",
+    "lodash.camelcase": "^4.3.0",
```

Comment:
Some projects I've worked on don't like 3rd party libs for this sort of thing. I guess we're ok with it?

Reply:
I hope so, I've been sneaking in lodashes everywhere:
```
$ git blame yarn.lock | grep "lodash" | cut -d"(" -f2 | cut -d" " -f1 | sort | uniq -c | sort -nr

    111 Santiago
     12 ludamad
      8 spypsy
      6 PhilWindle
      5 Leila
      4 Jan
```